tests/ui/test_discopane.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+from pages.desktop.discopane import DiscoveryPane, DiscoveryPanePage, AboutAddons
+
+@pytest.fixture
+def firefox_profile(base_url, firefox_profile, discovery_pane_url):
+    firefox_profile.set_preference('extensions.webapi.testing', True)
+    firefox_profile.set_preference('extensions.webservice.discoverURL', discovery_pane_url)
```

Comment:
We should include a comment regarding why these preferences are needed. The latter is fairly self-explanatory, but not the former.